### PR TITLE
BaSP-TG-29: super admin

### PR DESCRIPTION
**Pull request by Johsua30**
The feature corresponds to the screens for the creation, edition, deletion and for viewing a table with the super admins.
Edit and Add are separated in different paths

### CODE REVIEW

File: src/Components/SuperAdmins/App.jsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+      const data = await response.json();
+      setSuperAdmins(data.data);
+    } catch (error) {
+      console.error(error);
+    }
+  }, []);
+
+  const deleteSuperA = async (_id) => {
+    try {
+      const response = await fetch(`http://localhost:8000/api/super-admins/${_id}`, {
```
Comment (agusting19) [inline src/Components/SuperAdmins/App.jsx:19]: The fetch method has to be with the REACT_APP_API_URL, as in line 14

File: src/Components/SuperAdmins/App.jsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        method: 'DELETE'
+      });
+      console.log('response', response);
+      confirm(`WARNING!\n Are you sure you want to delete this super admin?`);
```
Comment (agusting19) [inline src/Components/SuperAdmins/App.jsx:23]: confirm need to be above the fetch, if the confirm is true, make the fetch

File: src/Components/SuperAdmins/App.jsx [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    } catch (error) {
+      alert(`Error\n${error}`);
+      console.error(error);
+    }
+    setSuperAdmins([...superAdmins.filter((listItem) => listItem._id !== _id)]);
```
Comment (agusting19) [inline src/Components/SuperAdmins/App.jsx:28]: it has to be inside the Try method, after the confirm and fetch

**Review by agusting19 — COMMENTED** [review]
You have to make little fixes

**Comment by AgustinCarthery** [comment]
## Testing Results 
Tested locally on https://alexis-trackgenix-app-git-feature-tg-29super-admin-basp-m2022.vercel.app/

**Test Results: Failed**  ⚠️

Bugs found: 

1. Super admins list does not include seeder data on it, data must be required for testing purposes of the feature 
Screen Shot 2022-06-01 at 16 43 02

2. Improper exit button position, button should be located on bottom left next to "Save" button and should contain "Cancel" txt instead. 
Screen Shot 2022-06-01 at 16 39 45

3. Failed fetch on onSubmit, adding a super-admin feature failed when hitting on "Save" btn. 
Screen Shot 2022-06-01 at 16 42 28

4. Active property on super-admin is not being required on FE html, please refer to BE validation on this line https://github.com/BaSP-m2022/alexis-trackgenix-server/blob/ede423c7299572aa5cd33bcd4423bec7750dfe2e/src/validation/super-admins.js#L11

Screen Shot 2022-06-01 at 17 08 35

Screen Shot 2022-06-01 at 17 08 30

**Notes: due to not having seed data on super-admin, Edit and Delete methods were not tested**

**Review by AgustinCarthery — APPROVED** [review]
Nice WORK 🥳

**Review by iaracriscenti — APPROVED** [review]

**Review by NicolasFontana — APPROVED** [review]

**Pull request merged**